pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -904,6 +1065,25 @@ func GatherServiceAccounts(i *Gatherer) func() ([]record.Record, []error) {
 	}
 }
 
+func getAllNamespaces(i *Gatherer) (*corev1.NamespaceList, error) {
+	ns, ok := i.ctx.Value(contextKeyAllNamespaces).(*corev1.NamespaceList)
```

Comment:
Namespaces are used in two Gatherers. This will try to cache them into context (one run of Gather) to save some time.